_docs/contracts.md

```diff
@@ -113,6 +113,11 @@ Restrictions for libraries in comparison to contracts:
 
 (these might be lifted at a later point)
 
+Common pitfalls for libraries
+
+* The value for `msg.sender` will be that of the contract which is calling the library function.  (If A calls contract B which internally calls library C, then within the function call of library C, `msg.sender` will be the address of contract B).
```

Comment:
Please take a look at my revision.  I took your paragraph and reworded it a bit to better fit the context.